src/test/java/com/thindeck/dynamo/DyTasksTest.java

```diff
@@ -68,8 +76,106 @@ public void getTask() throws Exception {
     }
 
     /**
+     * DyTask can add a task with attributes in NULL.
+     * @throws Exception In case of error.
+     */
+    @Test
+    @Ignore
+    public void addTaskNullAttributes() throws Exception {
+        final Repo repo = new MkRepo();
+        final Region region = DyTasksTest.region(repo.name());
+        final DyTasks tasks = new DyTasks(region, repo);
+        final Task task = tasks.add(DyTasksTest.CMD, null);
+        MatcherAssert.assertThat(
+            task.command(),
+            Matchers.equalTo(DyTasksTest.CMD)
+        );
+    }
+
+    /**
+     * DyTask can add a task without attributes.
+     * @throws Exception In case of error.
+     */
+    @Test
+    @Ignore
```

Comment:
@krzyk Because this test fail with the add method. This method was already implemented before that i take this task and the bug was reported https://github.com/yegor256/thindeck/issues/492